packages/additionalapi/src/database/AdditionalQueries.ts

```diff
@@ -25,15 +25,15 @@ export class AdditionalQueries {
         let query = ""
         try {
             if (nodeIdList.length === 0) {
-                return { status: 401, query: "query", queryResult: [] }
+                return { status: HttpSuccessCodes.NoContent, query: "query", queryResult: [] }
             }
             // TODO Currently only gives the node id's, should give full node.
             query = makeQueryNodeTreeForIdList(nodeIdList, depthLimit)
-            return { status: 410, query: query, queryResult: await this.context.dbConnection.query(query) }
+            return { status: HttpSuccessCodes.Ok, query: query, queryResult: await this.context.dbConnection.query(query) }
```

Comment:
This was `410` before, not `Ok`. On Purpose?

Reply:
Yes, it is on purpose. Wile changing the numbers to the enumeration values I checked all the status codes. Useful exercise since a number of them were incorrect.